packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 * Add iOS unit and UI integration test targets.
 * Exclude arm64 simulators in example app.
+* Remove references to the Android V1 embedding.
```

Comment:
This plugin isn't deprecated, so you should covert the NEXT to a version bump at this point.

Reply:
done